forte/data/ontology/top.py

```diff
@@ -1168,12 +1170,74 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    def __init__(
+        self,
+        pack: PackType,
+        modality: str,
+        payload_idx: int,
+    ):
+        self.payload_idx = payload_idx
+        self.modality = modality
+        super().__init__(pack)
+        self.cache = []
+        self.meta = {}
```

Comment:
Here the `meta` info is a bit too generic. Actually, we can use the ontology system to store `meta`, so that we can have structures on the meta data system too. This way users can define new data meta information as they wish. For example, one could develop more professional meta ontology type (just an example I quickly make up):

```json
{
  "entry_name": "ImageFormat",
  "parent_entry": "Meta",
  "description": "defines meta data for image format",
  "attributes": {
     "mime": "str",
      "type_code": "str",
      "type_identifier": "str",
      "standard": "str"
  }
}
```

```python
meta = ImageFormat()
meta.mime = "image/jpeg"
meta.type_code = "jpeg"
```

Reply:
Do you mean create a meta ontology class inherits from `Entry`? Is it independent from this [Meta](https://github.com/asyml/forte/blob/ac34f102ebb3319c1e23f2f81e20a7e9e6c5e60d/forte/data/data_pack.py#L67)

Reply:
yeah, the `Meta` class you referred is a very old implementation and not frequently used, at one point it can be deprecated.

Reply:
created a new `Meta` class